docs/openapi.yaml

```
@@ -2602,4 +2732,156 @@ components:
+          example: 1024768
+          description: File size in bytes for media messages
+        created_at:
+          type: string
+          format: date-time
+          example: '2024-01-15T10:30:00Z'
+          description: Record creation timestamp
+        updated_at:
+          type: string
+          format: date-time
+          example: '2024-01-15T10:30:00Z'
+          description: Record last update timestamp
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)